### PR TITLE
NEW show product label on inventory

### PR DESCRIPTION
**Pull request by lvessiller-opendsi**
NEW show product label on inventory

- in list and card
image

image

- add all paramaters for getNomUrl() and fix declaration compatibility for getNomUrl method of extended "ProductFournisseur" class

### CODE REVIEW

File: htdocs/fourn/class/fournisseur.product.class.php [hunk 1]
```diff
@@ -1091,12 +1091,15 @@ public function displayPriceProductFournisseurLog($productFournLogList = array()
...
+	 *  @param	int		$maxlength					Maxlength of ref
 	 *  @param  int     $save_lastsearch_value    	-1=Auto, 0=No save of lastsearch_values when clicking, 1=Save lastsearch_values whenclicking
+	 *  @param	int		$notooltip					No tooltip
+	 *  @param	int		$add_label					0=Default, 1=Add label into string, >1=Add first chars into string
+	 *  @param	string	$sep						' - '=Separator between ref and label if option 'add_label' is set
+	 *  @param  string  $morecss            		''=Add more css on link
 	 *	@return	string								String with URL
 	 */
-	public function getNomUrl($withpicto = 0, $option = '', $notooltip = 0, $morecss = '', $save_lastsearch_value = -1)
+	public function getNomUrl($withpicto = 0, $option = '', $maxlength = 0, $save_lastsearch_value = -1, $notooltip = 0, $add_label = 0, $sep = ' - ', $morecss = '')
```
Comment (eldy) [inline htdocs/fourn/class/fournisseur.product.class.php:1102]: Changing position of parameters will break the compatibility.

Comment (lvessiller-opendsi) [reply htdocs/fourn/class/fournisseur.product.class.php:1102]: @eldy 
I know that it's dangerous but getNomUrl method seems not to be used with the 3rd parameter.
I prefer to rewrite this method now with sames paramaters as Product class.
Do you think we can keep this method incompatible with Product class ?

**Comment by lvessiller-opendsi** [comment]
@eldy 
I resolved all conflits.

**Comment by lvessiller-opendsi** [comment]
@eldy 
I completed the Changelog file

**Pull request merged**